common/persistence/src/main/java/app/coronawarn/datadonation/common/persistence/domain/AndroidId.java

```diff
@@ -7,6 +7,10 @@
 @Table("android_id")
 public class AndroidId {
 
+  public AndroidId(String id) {
+    setId(id);
+  }
```

Comment:
by declaring this constructor, the default (empty) constructor is gone... does everything still work? All the DB magic?